MAINTAINERS.rst

```diff
@@ -85,17 +85,9 @@ Pre-release test plan
 Building and uploading the release wheels
 -----------------------------------------
 
-You need the zephyr-project PyPI credentials for the 'twine upload' command. ::
-
-  git clean -ffdx
-  pip3 install --upgrade build twine
-  pyproject-build
-  twine upload -u zephyr-project dist/*
-
-The 'git clean' step is important. We've anecdotally observed broken wheels
-being generated from dirty repositories.
-
-Check out [packaging.python.org](https://packaging.python.org/en/latest/tutorials/packaging-projects/#generating-distribution-archives) for more detailed instructions.
+Creating Pypi releases is done automatically from Github. After publishing
+a release on Github a release build is packaged and uploaded with the
+version specified in pyproject.toml.
```

Comment:
might want to keep the documentation for manual package upload process as a backup, while also mentioning that it is normally automated and manual uploads are not necessary.